commands/combat.py

```diff
@@ -230,31 +237,34 @@ def func(self):
         elif not shootable:
             caller.msg(f"{target.get_display_name(caller)} is too far away.")
             return
+        #elif too_close:
+        #    caller.msg(f"{target.get_display_name(caller)} is too close.")
+        #    return
```

Comment:
This code is commented out, is it intended?

Reply:
Yeah, I was too tired last night to figure out how to determine the minimum range check on ranged weapons, so I threw that in as a reminder, but it looks like I forgot to add a TODO FIXME on it.